test/integration/document_collection_test.rb

```diff
@@ -34,18 +24,58 @@ class DocumentCollectionTest < ActionDispatch::IntegrationTest
     assert page.has_text?("Each regime page provides a current list of asset freeze targets designated by the United Nations (UN), European Union and United Kingdom, under legislation relating to current financial sanctions regimes.")
   end
 
-  test "renders contents with link to each collection group" do
+  test "renders a contents list for each content item for every document collection group" do
```

Comment:
Adds a contents list, with one list item per document collection group, if the group contains documents.